src/averageAges.js

```diff
@@ -15,11 +15,13 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const peopleFiltered = century ? people.filter(a => {
+    const validateCentury = Math.ceil(a.died / 100) === century;
+
+    return validateCentury && a.sex === 'm';
+  }) : people.filter(a => a.sex === 'm');
```

Comment:
I love ternary, too, but this does look complicated
Perhaps, you could replaced it with `if` or move some operation off the main function